glean-core/src/lib.rs

```diff
@@ -128,7 +128,7 @@ impl Glean {
     ///
     /// TODO: (Verify this is correct):
     /// If the ping currently contains no content, it will not be sent.
```

Comment:
While we're at it: Can you adjust this comment and also mention what the return values mean?

Something like:
```
Returns true if a ping was sent, false otherwise.
Returns an error if collecting or writing the ping to disk failed.
```